hugegraph-server/pom.xml

```diff
@@ -236,13 +236,11 @@
                 <groupId>io.grpc</groupId>
                 <artifactId>grpc-netty</artifactId>
                 <version>${grpc.version}</version>
-                <scope>provided</scope>
```

Comment:
any context with the modification?

Reply:
Dependencies marked as provided scope are available during compilation and testing, but they will not be included in the final WAR or JAR package.

If you use provided scope here, an error class not found will occur when running hstore with pd-client related code.